Makefile

```diff
@@ -72,6 +72,14 @@ test: manifests generate fmt vet envtest ## Run tests.
 test-e2e:
 	go test ./test/e2e/ -v
 
+.PHONY: test-parallel
+test-parallel: ginkgo manifests generate fmt vet envtest ## Run tests in parallel
```

Comment:
Should `ginkgo manifests generate fmt vet envtest` be pulled into its own target?

If so, is it really necessary to have it before every test run? Or could we just document that `x-new-target` needs to be run before tests if there are "x, y, or z types of code changes"?